crates/adapters/src/static_compile/seroutput.rs

```diff
@@ -563,7 +563,8 @@ where
             val: None,
             phantom: PhantomData,
         };
-        result.skip_zero_keys();
+        debug_assert!(!result.cursor.key_valid() || !result.weight() != 0);
```

Comment:
I do not like this expression: `!result.weight() != 0`

Reply:
You were probably looking at an intermediate commit.